packages/blueprints-integration/src/context.ts

```diff
@@ -232,6 +232,10 @@ export interface ISyncIngestUpdateToPartInstanceContext extends IRundownUserCont
 	updatePartInstance(props: Partial<IBlueprintMutatablePart>): IBlueprintPartInstance
 }
 
+export interface IRemoveOrphanedPartInstanceContext extends IRundownUserContext {
```

Comment:
Should this be a UserContext?
Does it make sense to be able to show the user notes from uses of this?

Reply:
I think it might make sense to push something like "X has been orphaned, this may cause Y effect", depending on the installation specifics. e.g. Orphaning means that some metadata about the show won't be recorded. e.g. "You may want to reload your script to ensure [Some Media Server] is happy"